src/utils/__tests__/csvColumnAnalysis.test.ts

```
@@ -20,11 +22,13 @@ describe('csvColumnAnalysis', () => {
     it('detects exact column matches', () => {
       const headers = ['title', 'status', 'company', 'artist'];
       const result = analyzeCSVColumns(headers);
+
       
       expect(result.detectedColumns).toHaveLength(4);
       expect(result.detectedColumns[0]).toEqual({
         csvHeader: 'title',
         mappedTo: 'title',
+        confidence: 'exact',
```

Biome (1.9.4)

[error] This property value named confidence is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named confidence


(lint/suspicious/noDuplicateObjectKeys)